scriptoocloud/src/main/webapp/WEB-INF/views/admin/user/list.html

```diff
@@ -1,47 +1,94 @@
-<html layout:decorate="~{layouts/main}">
-  <head>
+<html lang="en"
+      xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout=xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
```

Comment:
that looks wrong; why are there two namespace/prefix definitions?

Reply:
also, is there a reason you are using a custom layout dialect here and not the default?